Mobile Buy SDK/Mobile Buy SDK Tests/BUYRequestOperationTests.m

```diff
@@ -332,7 +332,7 @@ - (void)testCancellationDuringPolling
 	
 	[self.queue addOperation:operation];
 	
-	[self createExpectationDelay:1.0 block:YES];
+	[self createExpectationDelay:10.0 block:YES];
```

Comment:
This adds 10 seconds to the test duration. It's not the same as waiting. Did you happen to test with any smaller values?